torch/csrc/api/include/torch/nn/modules/container/sequential.h

```diff
@@ -107,7 +107,7 @@ class SequentialImpl : public Cloneable<SequentialImpl> {
   explicit SequentialImpl(torch::OrderedDict<std::string, AnyModule>&& ordered_dict) {
     modules_.reserve(ordered_dict.size());
     for (auto& item : ordered_dict) {
-      push_back(std::move(item.key()), std::move(item.value()));
```

Comment:
Here `std::move` copies `Key` as `torch::OrderedDict::Item` has no overloaded method to get non-const reference to `Key` (which is quite reasonable as otherwise it could be possible to accidentally rewrite key without updating `std::unordered_map<Key, size_t> index_` of `OrderedDict` which in result would break it).